swizzle/provision.py

```diff
@@ -77,6 +76,7 @@ def generate_inventory(node_state={}):
         return temporary inventory file path """
     inventory = {
         "etcd": [],
+        "primary_master": [],
```

Comment:
Why is this an array? Looks like there will only be 1 item.

Reply:
This is how ansible works. It doesn't know that there will only one item.